app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -50,38 +56,55 @@
     .row
       .one.column.alpha &nbsp;
       .two.columns
-        %span.two.columns Group Buy Unit Size
+        %span.two.columns 
+          = t "group_buy_unit_size"
         %span.two.columns {{ formattedValueWithUnitName( selectedUnitsProduct.group_buy_unit_size, selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Total Quantity Ordered
+        %span.two.columns
+          = t "total_qtt_ordered"
         %span.two.columns {{ formattedValueWithUnitName( sumUnitValues(), selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Max Quantity Ordered
+        %span.two.columns
+          = t "max_qtt_ordered"
         %span.two.columns {{ formattedValueWithUnitName( sumMaxUnitValues(), selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Current Fulfilled Units
+        %span.two.columns
+          = t "current_fulfilled_units"
         %span.two.columns {{ fulfilled(sumUnitValues()) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Max Fulfilled Units
+        %span.two.columns
+          = t "max_fulfilled_units"
         %span.two.columns {{ fulfilled(sumMaxUnitValues()) }}
       .one.column.omega &nbsp;
     %div{ :class => "eight columns alpha", 'ng-hide' => 'allFinalWeightVolumesPresent()' }
       %span{ :class => "eight columns alpha", style: 'color:red' }
-        WARNING: Some variants do not have a unit value
-
-  %hr.divider.sixteen.columns.alpha.omega
-
-  .controls.sixteen.columns.alpha.omega{ ng: { hide: 'RequestMonitor.loading || lineItems.length == 0' } }
-    %div.three.columns.alpha
-      %input.fullwidth{ :type => "text", :id => 'quick_search', 'ng-model' => 'quickSearch', :placeholder => 'Quick Search' }
-    = render 'admin/shared/bulk_actions_dropdown'
-    %div.seven.columns &nbsp;
-    = render 'admin/shared/columns_dropdown'
-
+        = t "bulk_management_warning"
```

Comment:
Just had a very quick look - did you remove these changes? I think this might be why the test is failing - as far as I can see the DropDownCtrl that you added in doesn't exist so I get this when I run it locally:
``` 
Error: [ng:areq] Argument 'DropDownCtrl' is not a function, got undefined
```
Then a whole load of problems rendering the page which could explain the failures?